hipblas/PKGBUILD

```diff
@@ -10,10 +10,18 @@ license=('MIT')
 depends=('hip' 'rocblas' 'rocsolver')
 makedepends=('cmake' 'gcc-fortran')
 _git='https://github.com/ROCmSoftwarePlatform/hipBLAS'
-source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz")
-sha256sums=('f0fdaa851971b41b48ec2e7d640746fbd6f9f433da2020c5fd95c91a7473d9e1')
+source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
+		"include-path.patch::https://github.com/ROCmSoftwarePlatform/hipBLAS/commit/84b593c7c6acf176b21841a0441801d5a254c62b.patch")
+sha256sums=('5e9091dc4ef83896f5c3bc5ade1cb5db8e1a6afc451dbba4da19d8a7ec2b6f29'
+	    	'34652acc94caad9cf9b22fcf7658b7c7a3fb4c1b517b3aa15e5cc35e973b4fc6')
```

Comment:
This is still not indented properly.

Reply:
Can you elaborate? Currently the quotation are lined up like in the rocm-info PKGBUILD, but do you want the quotation marks to line up with the parentheses? Or are you referring to the misaligned spaces that show up when viewing it on the github website but not in a text editor?

Reply:
> Or are you referring to the misaligned spaces that show up when viewing it on the github website but not in a text editor?

The github website is showing the number of spaces just fine. I'm afraid you may have a setting such as 1 tab = 2 spaces or something unconventional. Please check your text editor settings again, until we see the proper diff. You can also check the diff local by typing in `git diff` or after the commit: `git diff HEAD~1`.